contents/blog/posthog-vs-statsig.mdx

```diff
@@ -116,11 +116,11 @@ Both PostHog and Statsig offer the functionality you expect from product analyti
   <ComparisonRow column1={true} column2={true} feature="Group analytics" description="Track metrics at a company level" />
   <ComparisonRow column1={true} column2={true} feature="Funnels" description="Track users through a sequence of events" />
   <ComparisonRow column1={true} column2={true} feature="Retention analysis" description="Visualize which users stay, for how long" />
-  <ComparisonRow column1={true} column2={false} feature="User paths" description="Track user flows and where they drop-off" />
+  <ComparisonRow column1={true} column2="In beta" feature="User paths" description="Track user flows and where they drop-off" />
   <ComparisonRow column1={true} column2={false} feature="Correlation analysis" description="Suggested events and properties that lead to success or failure" />
   <ComparisonRow column1={true} column2={false} feature="Lifecycle analysis" description="Understand who is dormant, churning, and thriving" />
   <ComparisonRow column1={true} column2={false} feature="Stickiness insights" description="See how many times users perform an event in a period of time." />
-  <ComparisonRow column1={true} column2={false} feature="Formulas" description="Use custom formulas to calculate unique insights" />
+  <ComparisonRow column1={true} column2={true} feature="Formulas" description="Use custom formulas to calculate unique insights" />
```

Comment:
Hey @akinolugbade . Thank you for updating these. I had a look through Statsigs product and docs but couldnt find more information on both these features. Would you mind sharing a link to the docs? Or screenshot in the product of how to do these?

Reply:
Perfect, thank you!